packages/workbox-routing/src/lib/router.js

```diff
@@ -13,11 +13,12 @@
  limitations under the License.
 */
 
-import Route from './route';
-import {isArrayOfClass, isInstance} from '../../../../lib/assert';
+import {isArrayOfInterface, isInterface} from '../../../../lib/assert';
 import logHelper from '../../../../lib/log-helper.js';
 import normalizeHandler from './normalize-handler';
 
+const ROUTE_INTERFACE = ['match', 'handler', 'method'];
```

Comment:
I'm not a huge fan of having this manually defined and it being defined away from the class it represents.